e2e/packages/contracts/test/StaticArrayLength.t.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { IStoreErrors } from "@latticexyz/store/src/IStoreErrors.sol";
+import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
+import { MudTest } from "@latticexyz/world/test/MudTest.t.sol";
+
+import { StaticArray } from "../src/codegen/index.sol";
+import { IWorld } from "../src/codegen/world/IWorld.sol";
+
+/**
+ * @title Wrapper
+ * @dev For testing that the getItemValue call properly reverts
+ */
+contract Wrapper {
+  function getItemValue(address worldAddress, uint256 _index) public {
+    StoreSwitch.setStoreAddress(worldAddress);
+
+    StaticArray.getItemValue(_index);
+  }
+}
+
+contract StaticArrayLengthTest is MudTest {
+  function testLength() public {
```

Comment:
is there a test somewhere that tests that if an element is set that value is returned instead of the default `0`? (maybe somewhere in the existing test suite?)

Reply:
There some some in CLI, maybe we should move these here? https://github.com/latticexyz/mud/blob/main/packages/cli/contracts/test/Tablegen.t.sol#L70-L77